app/assets/stylesheets/modules/top_nav.scss

```diff
@@ -27,6 +39,8 @@
       font-size: 0.9em;
       margin-right: 1.5em;
       padding-bottom: 0.3em;
+      font-weight: bold;
+      color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.